squire_lib/src/players/mod.rs

```diff
@@ -45,6 +47,19 @@ pub struct Player {
 }
 
 impl Player {
+    /// Returns a string of a player name and, game name
+    pub fn all_names(&self) -> String {
+        match &self.game_name {
```

Comment:
In Rust, you can add [match guards](https://doc.rust-lang.org/rust-by-example/flow_control/match/guard.html) to the branches of your match statements, so you can run this into something like:
```suggestion
        match &self.game_name {
            Some(game_name) if self.name.eq(game_name) => self.name.clone(),
            Some(game_name) => format!("{} ({game_name})", self.name),
            None => self.name.clone(),
        }
```